src/modules/admin/components/users/UserAuditPage.tsx

```diff
@@ -15,14 +17,28 @@ import useDebouncedState from '@/hooks/useDebouncedState';
 import useSafeParams from '@/hooks/useSafeParams';
 import ClientAccessSummaryTable from '@/modules/admin/components/users/ClientAccessSummaryTable';
 import EnrollmentAccessSummaryTable from '@/modules/admin/components/users/EnrollmentAccessSummaryTable';
+import UserAuditHistory from '@/modules/admin/components/users/UserAuditHistory';
 import { useUser } from '@/modules/dataFetching/hooks/useUser';
 import ClientTextSearchInput from '@/modules/search/components/ClientTextSearchInput';
 import CommonSearchInput from '@/modules/search/components/CommonSearchInput';
 import { AdminDashboardRoutes } from '@/routes/routes';
 
-type EntityType = 'clients' | 'enrollments';
+type PageEntityType = 'access' | 'edits';
```

Comment:
I'm not happy with this name `PageEntityType`, it would be much better to change it to something clearer, let me know if you have a good suggestion! I will keep thinking on it too

Reply:
Maybe `UserAuditType`?

Reply:
Or `UserHistoryType`